classes/Mail.php

```diff
@@ -58,7 +58,7 @@ class MailCore extends ObjectModel
                 'validate' => 'isEmail',
                 'copy_post' => false,
                 'required' => true,
-                'size' => 126,
+                'size' => 255,
```

Comment:
When you apply changes to the model, make sure you update `db_structure.sql` & the equivalent upgrade file or it will remain invalid.

Reply:
I already did. The reason why I had to change this file now, is that the email validator was not updated and that caused an exception. There were several places to update the size without in the database files, and this place was missing. :)